teuthology/task/install/rpm.py

```diff
@@ -144,6 +145,10 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
         log.info('Package version is %s', builder.version)
         builder.install_repo()
 
+    if repos_only:
```

Comment:
This code should be executed only if repos is specified, not valid for for builder branch for 'if' above?

Reply:
In my view, this could also be useful in the builder case. That's why I implemented it in this way.